docker/docker-compose.yml

```
@@ -65,7 +65,7 @@ services:
 
   rest:
     container_name: supabase-rest
-    image: postgrest/postgrest:v8.0.0
+    image: postgrest/postgrest:v9.0.0-a1
```

This one is older than `v9.0.0-a1`(there was an issue with the version naming scheme).
```suggestion
    image: postgrest/postgrest:v8.0.0.20211102
```